sos/report/plugins/sunbeam.py

```diff
@@ -94,7 +94,8 @@ def setup(self):
 
             self.add_copy_spec([
                 f"{sb_snap_homedir}/*.log",
-                f"{sb_snap_homedir}/etc/*/*.log",
+                f"{sb_snap_homedir}/etc/*/*/*.log",
+                f"{sb_snap_homedir}/etc/*/*/terraform.tfvars.json",
```

Comment:
I'm reading in some places, that some users may store the following two tf variables:

```
"aws_access_key": "accesskey",
"aws_secret_key": "secretkey"
```

It seems it's discouraged, as it should be, and users could use instead environment variables. But just to be on the safe side, should we add these two possible entries to be obfuscated via postproc?